packages/paste-core/components/form-pill-group/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {CustomizationProvider} from '@twilio-paste/customization';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {useFormPillState, FormPillGroup, FormPill} from '../src';
+import {Basic, SelectAndRemove, CustomFormPillGroup} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+const CustomElementFormPillGroup: React.FC = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup
+        {...pillState}
+        element="CUSTOM_PILL_GROUP"
+        data-testid="form-pill-group"
+        aria-label="Your favorite sports:"
+      >
+        <FormPill {...pillState} element="CUSTOM_PILL" data-testid="form-pill">
+          Tennis
+        </FormPill>
+      </FormPillGroup>
+    </form>
+  );
+};
+
+describe('FormPillGroup', () => {
+  // Verifies that the correct aria attributes and semantics are met
+  describe('Rendered shape', () => {
+    it('should render correctly', () => {
+      const {getByTestId, getByText} = render(<Basic />);
+      expect(getByText('Tennis')).toBeDefined();
+
+      const group = getByTestId('form-pill-group');
+      expect(group.getAttribute('aria-label')).toBe('Your favorite sports:');
+      expect(group.tagName).toBe('DIV');
+      expect(group.getAttribute('role')).toBe('listbox');
+
+      const pill = getByTestId('form-pill');
+      expect(pill.tagName).toBe('BUTTON');
+      expect(pill.getAttribute('role')).toBe('option');
```

Comment:
Once we have `aria-selected` reflecting the selected state of a pill, it would be nice to have a test that ensures we're selecting pills correctly.

Do you also think it would be worthwhile just having one test that moves focus around using the arrow keys to ensure the keyboard nav continues to work? I know that's quite well tested in Composite, but we might want to test our implementation just in case we mess something up in the future

Reply:
Added a bunch of tests for this behavior.